tests/snmpsim/audiocodes_mp118.snmprec

```diff
@@ -0,0 +1,1910 @@
+1.3.6.1.2.1.1.1.0|4|Product: GW 8 FXS;SW Version: 6.60A.309.001
```

Comment:
The initial files shouldn't have a variant specified. It's only additional files that need to do that.

Reply:
I can change it easily but the devices are really different so it would be better to create some basic test file (with only sysDescr and sysObjectID) and already create a variant with its proper name. What would you say ?